### PR TITLE
Added Phishing Website Detection project (#61)

### PR DESCRIPTION
**Pull request by stiwari-ds**
# Pull Request for DL-Simplified 💡

## Issue Title :  Phishing Website Detection

- **Info about the related issue (Aim of the project)** :  Training a deep neural network model to classify websites as legitimate or phishing using 30 extracted features based on URL, port, redirect, etc.
- **Name:**  Siddhant Tiwari
- **GitHub ID:**  stiwari-ds
- **Email ID:**  siddhant.tiwari.ds@gmail.com
- **Identify yourself: (Mention in which program you are contributing in. Eg. For a JWOC 2022 participant it's, `JWOC Participant`)**  SWOC Participant

Closes: #61 

### Describe the add-ons or changes you've made 📃
1. EDA - data cleaning; feature selection for dimensionality reduction
2. Baseline - simple ML models (linear and non-linear) for baseline score
3. Deep neural network - trained multi-layer perceptron to beat the baseline score; tuned hyperparameters further using halving grid-search to maximize accuracy.

## Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist: ☑️

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Review by 072arushi — APPROVED** [review]
Superb work and a great approach. The accuracy attained is pretty high too.

**Comment by stiwari-ds** [comment]
> Superb work and a great approach. The accuracy attained is pretty high too.

Thanks!

**Review by abhisheks008 — APPROVED** [review]
Superb!🔥💥 
Your PR is approved and ready to be merged.
@stiwari-ds

**Comment by stiwari-ds** [comment]
@abhisheks008 Thanks! Awesome to have my first merged PR 😁
